lm_eval/__main__.py

```diff
@@ -142,6 +142,12 @@ def parse_eval_args() -> argparse.Namespace:
         metavar="CRITICAL|ERROR|WARNING|INFO|DEBUG",
         help="Controls the reported logging error level. Set to DEBUG when testing + adding new task configurations for comprehensive log output.",
     )
+    parser.add_argument(
```

Comment:
might be worth adding an assert in evaluate() that checks we haven't set predict_only and not log_samples?

Reply:
Alternatively, we can just auto-set log_samples if predict_only is used to decrease the cognitive load on the user